pytorch_toolkit/ote/ote/modules/trainers/mmdetection.py

```diff
@@ -16,12 +16,12 @@
 
 import json
 import logging
-import subprocess
 import tempfile
 
 from mmcv.utils import Config
 
 from ote import MMDETECTION_TOOLS
+from ote.utils.misc import run_through_shell
```

Comment:
what is it for?

Reply:
It was used for clustering script running. Now it is not required.

Reply:
Fixed